generation/maisi/scripts/inference.py

```diff
@@ -132,6 +132,10 @@ def main():
 
     # check the format of inference inputs
     config_infer_dict = json.load(open(args.inference_file, "r"))
+    # override num_split if asked
+    if "autoencoder_tp_num_splits" in config_infer_dict:
```

Comment:
Is there any special reason not directly using `num_splits` in the config file and here?